web/app/mu-plugins/ppj/ppj-functions.php

```diff
@@ -360,3 +348,14 @@ function markCurrentlySelectedAncestorMenuItem($menuItems) {
     }
     return $modifiedMenuItems;
 }
+
+/**
+ * @param $str
+ *
+ * Simple helper function to remove unicode characters using regex matching.
+ *
+ * @return mixed
```

Comment:
We already discussed removing the method `removeUnicodeCharacters`. It doesn't look like it's used, and it's not clear exactly what it does (since we're not intimately familiar with the Unicode range x00–x7F. Please remove.

Reply:
This was done in this commit: https://github.com/ministryofjustice/wp-ppj/pull/26/commits/5b7b52ab1ccd6273931fabe8b8ce0eb6ebb6ed39

Reply:
So let's go ahead and remove this unused method.

Reply:
Good point. It wasn't completely removed in https://github.com/ministryofjustice/wp-ppj/commit/5b7b52ab1ccd6273931fabe8b8ce0eb6ebb6ed39